pkg/rpcclient/rpc_test.go

```diff
@@ -1375,6 +1375,80 @@ var rpcClientTestCases = map[string][]rpcClientTestCase{
 			},
 		},
 	},
+	"getrawnotarytransaction": {
+		{
+			name: "positive",
+			invoke: func(c *Client) (any, error) {
+				hash, err := util.Uint256DecodeStringLE("ad1c2875de823a54188949490e2d68580fd070fcc5ff409609f478d23d12355f")
+				if err != nil {
+					panic(err)
+				}
+				return c.GetRawNotaryTransaction(hash)
+			},
+			serverResponse: `{"id":1,"jsonrpc":"2.0","result":{"hash":"0xad1c2875de823a54188949490e2d68580fd070fcc5ff409609f478d23d12355f","size":61,"version":0,"nonce":3,"sender":"Nhfg3TbpwogLvDGVvAvqyThbsHgoSUKwtn","sysfee":"0","netfee":"0","validuntilblock":123,"attributes":[{"nkeys":1,"type":"NotaryAssisted"}],"signers":[{"account":"0xb248508f4ef7088e10c48f14d04be3272ca29eee","scopes":"None"}],"script":"QA==","witnesses":[{"invocation":"AQQH","verification":"AwYJ"}]}}`,
+			result: func(c *Client) any {
+				return &transaction.Transaction{}
+			},
+		},
+		{
+			name: "positive verbose",
+			invoke: func(c *Client) (any, error) {
+				hash, err := util.Uint256DecodeStringLE("ad1c2875de823a54188949490e2d68580fd070fcc5ff409609f478d23d12355f")
+				if err != nil {
+					panic(err)
+				}
+				return c.GetRawNotaryTransactionVerbose(hash)
+			},
+			serverResponse: `{"id":1,"jsonrpc":"2.0","result":{"hash":"0xad1c2875de823a54188949490e2d68580fd070fcc5ff409609f478d23d12355f","size":61,"version":0,"nonce":3,"sender":"Nhfg3TbpwogLvDGVvAvqyThbsHgoSUKwtn","sysfee":"0","netfee":"0","validuntilblock":123,"attributes":[{"nkeys":1,"type":"NotaryAssisted"}],"signers":[{"account":"0xb248508f4ef7088e10c48f14d04be3272ca29eee","scopes":"None"}],"script":"QA==","witnesses":[{"invocation":"AQQH","verification":"AwYJ"}]}}`,
+			result: func(c *Client) any {
+				return &transaction.Transaction{}
+			},
+		},
+	},
+	"getrawnotarypool": {
+		{
+			name: "empty pool",
+			invoke: func(c *Client) (any, error) {
+				return c.GetRawNotaryPool()
+			},
+			serverResponse: `{"id":1,"jsonrpc":"2.0","result":{}}`,
+			result: func(c *Client) any {
+				return &result.RawNotaryPool{
+					Hashes: map[util.Uint256][]util.Uint256{},
+				}
+			},
+		},
+		{
+			name: "nonempty pool",
+			invoke: func(c *Client) (any, error) {
+				return c.GetRawNotaryPool()
+			},
+			serverResponse: `{"id":1,"jsonrpc":"2.0","result":{"hashes":{"d86b5346e9bbe6dba845cc4192fa716535a3d05c4f2084431edc99dc3862a299":["bb0b2f1d5539dd776637f00e5011d97921a1400d3a63c02977a38446180c6d7c"]}}}`,
```

Comment:
Have you taken these hashes from our server response? They are in the wrong format, there should be `0x` prefix.

Reply:
Yep, that's because of https://github.com/nspcc-dev/neo-go/pull/3098#discussion_r1309895109

Reply:
Let's fix it.

Reply:
Fixed.